addons/account/models/partner.py

```diff
@@ -368,7 +370,7 @@ def _compute_has_unreconciled_entries(self):
 
     def mark_as_reconciled(self):
         self.env['account.partial.reconcile'].check_access_rights('write')
-        return self.sudo().with_context(company_id=self.env.company.id).write({'last_time_entries_checked': time.strftime(DEFAULT_SERVER_DATETIME_FORMAT)})
+        return self.sudo().write({'last_time_entries_checked': time.strftime(DEFAULT_SERVER_DATETIME_FORMAT)})
```

Comment:
you miss a `with_company` there, right?

Reply:
with_company(self.env.company.id) change la valeur de retour de self.env.company.id, ce qui ne change donc plus rien.

Reply:
Rien ne te garantit que le `env.company` du SUPERUSER est le même que le `env.company` du user courant ;)

Reply:
self.sudo() ne change pas l'utilisateur courant ;).

Reply:
Tu veux dire que self.sudo() ne change plus l'utilisateur courant :) Ok you get the point!